backend/clubs/management/commands/stress_test.py

```diff
@@ -0,0 +1,173 @@
+import asyncio
+import datetime
+import logging
+import random
+import time
+
+from asgiref.sync import sync_to_async
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db.models import Prefetch
+from django.utils import timezone
+from rest_framework.test import APIRequestFactory
+
+from clubs.models import ApplicationQuestion, Club, ClubApplication
+from clubs.views import UserViewSet
+
+
+class Command(BaseCommand):
+    help = """
+        Runs stress tests on Penn Clubs database to assess performance
+        upgrades on club submissions.
+        """
+
+    def setUp(self):
+        self.num_clubs = 10
+        self.num_users = 10
+        self.subset_size = 1
+        self.num_questions_per_club = 2
+        self.total_submissions = 1
+        self.club_prefix = "test_club_"
+        self.user_prefix = "test_user_"
+
+        self.uri = "/users/question_response/"
+        self.factory = APIRequestFactory()
+        self.view = UserViewSet.as_view({"post": "question_response"})
```

Comment:
This will create objects for real right? So we want to tear down?

Reply:
Yeah the `tearDown()` function handles that, but if it needs to be more explicit I can do that.